modules/gridmap/grid_map_editor_plugin.cpp

```diff
@@ -597,29 +597,31 @@ bool GridMapEditor::forward_spatial_input_event(Camera *p_camera, const Ref<Inpu
 			if (mb->get_button_index() == BUTTON_LEFT) {
 
 				if (input_action == INPUT_DUPLICATE) {
-
 					//paste
 					_duplicate_paste();
 					input_action = INPUT_NONE;
 					_update_duplicate_indicator();
 				} else if (mb->get_shift()) {
 					input_action = INPUT_SELECT;
-				} else if (mb->get_command())
+				} else if (mb->get_command()) {
 					input_action = INPUT_COPY;
-				else {
+				} else {
 					input_action = INPUT_PAINT;
 					set_items.clear();
 				}
-			} else if (mb->get_button_index() == BUTTON_RIGHT)
+			} else if (mb->get_button_index() == BUTTON_RIGHT) {
 				if (input_action == INPUT_DUPLICATE) {
-
 					input_action = INPUT_NONE;
 					_update_duplicate_indicator();
 				} else if (mb->get_shift()) {
 					input_action = INPUT_ERASE;
 					set_items.clear();
-				} else
+				} else {
 					return false;
+				}
+			} else {
```

Comment:
I've added this as going back to 2014 commits the indentation showed that this was the intent. Yet an `else` is also needed in the `BUTTON_RIGHT` branch to preserve the same behaviour as old code did not have the `if (mb->get_shift())` check.